index.js

```diff
@@ -67,6 +71,54 @@ function toggleColor(element) {
     }
 }
 
+header.addEventListener('click', function(){
+    if (header.style.color === 'black'){
+        header.style.color = 'red'
+    } else {
+        header.style.color = 'black'
+    }
+    
+})
+// didnt see toggleColor() when I originally wrote the function uptop. I'd basically take
+// that and send it into the listener method...please verify if the following would be correct.
+// addEventListner ('click',toggleColor)
 /***** Deliverable 2 *****/
 
+let likesElement = document.querySelector("#profile > p")
+let buttonElement = document.querySelector("#profile > button")
+buttonElement.addEventListener('click', function() {
+    likesInt = parseInt(likesElement.innerText.split(" ")[0],10)
+    likesInt += 1
+    likesElement.innerText = `${likesInt} Likes`
+})
+
 /***** Deliverable 3 *****/
+
+let formElement = document.querySelector("#new-animal-sighting-form")
+let sightingsList = document.querySelector("#animals")
+
+formElement.addEventListener("submit", function(event) {
+     event.preventDefault()
+
+     let speciesInput = event.target.species.value
+     let linkInput = event.target.link.value
+     let photoInput = event.target.photo.value
+     let descriptionInput = event.target.description.value
+     let animalId = sightingsList.children.length + 1
+
+     infoArray = [speciesInput, linkInput, photoInput, descriptionInput, animalId]
```

Comment:
This works, but instead of throwing these into an array you could also throw them into an object like

```javascript
const newData = {
        species: species,
        videoLink: videoLink,
        photo: photo,
        description: description
    }
```